tool-plugins/intellij/src/main/java/icons/BallerinaIcons.java

```diff
@@ -54,6 +54,18 @@
     public static final Icon PUBLIC_FIELD = createPublicFieldIcon(AllIcons.Nodes.Advice);
     public static final Icon PRIVATE_FIELD = createPrivateFieldIcon(AllIcons.Nodes.Advice);
 
+    /**
+     * Contains icons used for the ballerina diagram editor(referred by plugin.xml).
+     */
+    public static class Layout {
+        public static final Icon EDITOR_ONLY = IconLoader
+                .getIcon("/icons/layout/code.svg", BallerinaIcons.class); // 16x16
```

Comment:
Shall we break the lines after `=` to increase the readability? WDYT?

Reply:
Fixed.